### PR TITLE
add emoji 9 test

### LINKED ISSUE #10: Missing Unicode 9.0 "Rolling on the floor laughing" and more
This otherwise wonderful and highly useful regular expression seems to be missing a number of modern / new Emoji included in [Unicode 9](http://emojipedia.org/unicode-9.0/) / [Emoji 3.0](http://emojipedia.org/emoji-3.0/).  The missing emojis include the popular [Rolling on the floor laughing](http://emojipedia.org/rolling-on-the-floor-laughing/) (:rofl:) and [Nauseated Face](http://emojipedia.org/nauseated-face/) (:nauseated_face:), as well as others.

screenshot 2017-01-15 21 19 30

Any chance we can get these new ones added to the regex?

Thanks for an awesome library!

\- Joe

### PR DESCRIPTION
**Pull request by patrickkettner**
pulled from the latest [emoji-data.txt](http://unicode.org/Public/emoji/5.0/emoji-data.txt), in order to fix [emoji-regex#10](https://github.com/mathiasbynens/emoji-regex/issues/10)

### CODE REVIEW

**Comment by mathiasbynens** [comment]
Could you look into the failing test please? Thanks!

**Comment by mathiasbynens** [comment]
Did you edit this file manually? The idea of this repo is that [`npm run build`](https://github.com/mathiasbynens/regenerate-unicode-properties/blob/c33b350d82dd47bdfa43cf3a50f58341f0d5d664/package.json#L44) generates all the output, including `Binary_Property/Emoji.js`. So, to do what you want, [unicode-tr51](https://github.com/mathiasbynens/unicode-tr51) must be updated to use the latest data, after which this project can update its dependencies and run the build again.

**Comment by mxstbr** [comment]
It's dependencies all the way down...

**Comment by patrickkettner** [comment]
created PR for the correct change, and updated this to just be a test addition for level 9 emoji (that will obviously fail until the unicode-tr51 change it merged)

File: tests/tests.js [hunk 1]
```diff
@@ -9,6 +9,9 @@ test(t => {
 	t.true(
 		require('../Binary_Property/Emoji.js') instanceof regenerate
 	);
+	t.true(
+		require('../Binary_Property/Emoji.js').toRegExp().test('🤡')
```
Comment (mathiasbynens) [inline tests/tests.js]: Let’s use `'\u{1F921}'` instead so the test files remain ASCII-safe.

**Review by mathiasbynens — CHANGES_REQUESTED** [review]
Could you update the unicode-tr51 dependency to the latest version? Then this looks good to go 👍

File: tests/tests.js (hunk 1 above)
Comment (mathiasbynens) [reply tests/tests.js]:
```js
require('../Binary_Property/Emoji.js').toRegExp().test('\u{1F921}') // U+1F921 CLOWN FACE
```

**Comment by patrickkettner** [comment]
done

**Review by mathiasbynens — APPROVED** [review]

**Comment by mathiasbynens** [comment]
[Huh?](https://travis-ci.org/mathiasbynens/regenerate-unicode-properties/jobs/208726335#L899)

```
Creating Binary_Property/Emoji_Component.js…
module.js:472
    throw err;
    ^
Error: Cannot find module 'unicode-tr51/Emoji_Component.js'
```

File: package.json [hunk 2]
```diff
@@ -38,7 +38,7 @@
     "fs-extra": "^1.0.0",
     "jsesc": "^2.2.0",
     "unicode-9.0.0": "^0.7.0",
-    "unicode-tr51": "^7.0.2"
+    "unicode-tr51": "^8.0.0"
```
Comment (mathiasbynens) [inline package.json]: Could you update this to v8.0.1 explicitly please? 8.0.0 is missing `Emoji_Component.js`.

Comment (patrickkettner) [reply package.json]: done

**Pull request merged**